v0/destinations/serenytics/data/SerenyticsTrackConfig.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "destKey": "id",
+    "sourceKeys": "messageId",
+    "required": false
+  },
+  {
+    "destKey": "anonymous_id",
+    "sourceKeys": "anonymousId",
+    "required": false
+  },
+  {
+    "destKey": "event",
+    "sourceKeys": "event",
+    "required": true
+  },
+  {
+    "destKey": "context_ip",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "required": false
+  },
+  {
+    "destKey": "received_at",
+    "sourceKeys": "receivedAt",
+    "required": false
+  },
+  {
+    "destKey": "sent_at",
+    "sourceKeys": "sentAt",
+    "required": false
+  },
+  {
+    "destKey": "original_timestamp",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
```

Comment:
These two will be the same. Is that intentional?

Reply:
No, changed with `originalTimestamp`.